File: Aula04/triangulo/src/Ponto.java
public class Ponto {
    private double x, y;

    public Ponto(double x, double y){
        this.x = x;
        this.y = y;
    }

    public double getX(){
        return x;
    }

    public void setX(double x){
        this.x = x;
    }

    public double getY(){
        return y;
    }

    public void setY(double y){
        this.y = y;
    }

    public double distPontos(Ponto p1){
        double dist = Math.pow(this.getX() - p1.getX(), 2);
        dist += Math.pow(this.getY() - p1.getY(), 2);
        return Math.sqrt(dist);
    }
}

File: Aula06/Ponto.cpp
#include "Ponto.hpp"
#include <iostream>
#include <cmath>

Ponto::Ponto(const double& x, const double& y){
    this->x = x;
    this->y = y;
}
void Ponto::setX(const double& x){
    this->x = x;
}
void Ponto:: setY(const double& y){
    this->y = y;
}
double Ponto:: getX() const{
    return this->x;
}
double Ponto:: getY() const{
    return this->y;
}
double Ponto::distPontos(Ponto p1){
    double diff1 = this->getX() - p1.getX();
    double diff2 = this->getY() - p1.getY();
    double dist = pow(diff1, 2);
    dist += pow(diff2, 2);
    return sqrt(dist);
}